src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -128,7 +128,7 @@ void KviMenuBar::setupHelpPopup(QMenu * pop)
 	help->addSeparator();
 	pAction = help->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::HomePage)), __tr2qs("KVIrc Home&page"));
 	pAction->setData(KVI_INTERNALCOMMAND_KVIRC_HOMEPAGE);
-	if(KviLocale::instance()->localeName() == "ru")
+	if(QString::compare(KviLocale::instance()->localeName(), QString("ru"), Qt::CaseInsensitive))
```

Comment:
`QString::compare` does not return a bool of true if equal.  If you want to compare for string equality, you need to do `QString::compare(...) == 0`.